scripts/wallet_web_dev_start.sh

```diff
@@ -15,7 +15,7 @@ cd $ROOT/test/fixtures/wallet-web
 (source .env && docker-compose -f docker-compose-demo.yml -f docker-compose-server.yml -f docker-compose-web.yml down && docker-compose -f docker-compose-demo.yml -f docker-compose-server.yml -f docker-compose-web.yml up --force-recreate -d)
 
 echo "waiting for containers to start..."
-sleep 15s
+sleep 15
```

Comment:
After upgrading macOS to Monterey, this started throwing an error. As per documentation, the numeric value provided as a parameter is always in seconds and does not require specifying the suffix `s`. Let me know if they works on older macOS versions, which I think it should...

<img width="922" alt="image" src="https://user-images.githubusercontent.com/33902374/144755492-8d3cd254-9c75-4895-aac1-afc801ac2b5f.png">